app/views/orders/_service_desc.html.haml

```diff
@@ -19,5 +19,5 @@
     - if order_detail.stored_files.template_result.empty?
       = link_to 'Upload Order Form', order_order_detail_order_file_path(order_detail.order, order_detail), :class => 'btn'
     - else
-      = link_to('Uploaded Order Form', order_detail.stored_files.template_result.first.file.url)
+      = link_to "Uploaded Order Form", order_detail_first_template_result_path(order_detail)
```

Comment:
I think this might need to go through a different controller action. Where this goes (`OrderManagement::OrderDetailsController#template_results`) only loads purchased orders, so I'm getting a 404 on `Couldn't find Order with id=40 [WHERE `orders`.`facility_id` = 1 AND (ordered_at IS NOT NULL)]`